src/CSSBattle/8_ForkingCrazy/WeiPin.html

```diff
@@ -0,0 +1,90 @@
+<div class='fork'>
+    <div class='fdark1'></div>
+    <div class='g1'></div>
+    <div class='fdark2'></div>
+    <div class='g2'></div>
+    <div class='fdark3'></div>
+    <div class='g3'></div>
+    <div class='fdark4'></div>
+    <div class='handle'></div>
+  </div>
+  
+  <style>
+    body {
+      background: #6592CF;
+      margin: 0;
+      position: relative;
+    }
+    .fork {
+      position: 
+      width: 20px;
+      height: 150px;
+      background: #060F55;
+      border-radius: 0px 0px 70px 70px;
+      margin: 100 130 
+    }
+    .fdark1 {
```

Comment:
may want to abstract common properties into a class, for example fork, then for the specific additional differences (position) into a separate class / id  to make it more specific